StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetError.swift

```diff
@@ -62,66 +63,6 @@ public enum PaymentSheetError: Error, LocalizedError {
 }
 
 extension PaymentSheetError: CustomDebugStringConvertible {
-    /// A string that can safely be logged to our analytics service that does not contain any PII
-    public var safeLoggingString: String {
```

Comment:
This should never have been public (added [here](https://github.com/stripe/stripe-ios/pull/2729/files#diff-9481030907897f02a164f3ed394a5c2e138b927a1d13fe0bec49a70956599f67R70)) so I'm adding the `modifies public API` label.